app/build.gradle

```diff
@@ -2,6 +2,10 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+plugins {
+    id "com.jetbrains.python.envs" version "0.0.25"
```

Comment:
nit: please note that this was changed in the README and the sample app. You need to use "0.0.26" now.